sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -480,7 +480,7 @@ public async Task StartRecognizeContentWithMultiplePageArgument(string page1, st
             using var stream = FormRecognizerTestEnvironment.CreateStream(TestFile.InvoiceMultipageBlank);
             using (Recording.DisableRequestBodyRecording())
             {
-                operation = await client.StartRecognizeContentAsync(stream, new RecognizeContentOptions() { Pages = new List<string> { page1, page2 } });
+                operation = await client.StartRecognizeContentAsync(stream, new RecognizeContentOptions() { Pages = { page1, page2 } });
```

Comment:
Never knew a list property could be initialized like this.

Reply:
Me neither! lol. I learned from KV and their documentation. But in case our customers don't know, I added the link to the docs.ms 
it is pretty cool!